cockatrice/src/tab_deck_editor.cpp

```diff
@@ -988,6 +994,17 @@ void TabDeckEditor::actRemoveCard()
     if (!currentIndex.isValid() || deckModel->hasChildren(currentIndex))
         return;
     deckModel->removeRow(currentIndex.row(), currentIndex.parent());
+
+    DeckLoader *const deck = deckModel->getDeckList();
+    QString decklistUrlString;
+    if (deck->isEmpty()) {
+        aSaveDeck->setEnabled(false);
+        aSaveDeckAs->setEnabled(false);
+    }
+    else {
```

Comment:
Tiny code style error left ❤️

Reply:
I'll get on all that stuff right away!

Speaking of style errors, I noticed a lot of if/else statements omitting braces for single-statements which goes against the code style guide. Would there be any utility to refactoring them to match the style guide?

Reply:
Normally clang-format should catch and report those. It can also fix them with a simple command, that's why we use it. :)
Speaking about that, we still have no description setup on how to properly use it...

Maybe somebody else know what's up with the if/else style you mentioned.
Config got added and discussed in https://github.com/Cockatrice/Cockatrice/pull/3028 by the way.

In the meantime can you point to 1-2 examples?

Reply:
![image](https://user-images.githubusercontent.com/41343559/44998436-4f3c7300-af7b-11e8-9e22-509ee33de325.png)

This if/else block contradicts the Braces section in the contributing guide in that the braces are not on their own line.

![image](https://user-images.githubusercontent.com/41343559/44998461-7eeb7b00-af7b-11e8-9dd2-719f437c1349.png)

This also contradicts the Braces section of the Contributing guide since the braces are omitted.

I think maybe it's the contributing guide that should be updated to reflect the way the automatic code review bots enforce the code styling?